### PR TITLE
APR-1322 Website address

### PR DESCRIPTION
**Pull request by MakCom**

### CODE REVIEW

File: src/SFA.DAS.ApplyService.InternalApi.UnitTests/GatewayOrganisationChecksControllerTests.cs [hunk 1]
```diff
@@ -117,5 +120,21 @@ public void get_website_address_manually_entered_returns_no_value_when_not_prese
 
             Assert.IsNull(actualResult);
         }
+
+
+        [Test]
```
Comment (David-B-Read) [inline src/SFA.DAS.ApplyService.InternalApi.UnitTests/GatewayOrganisationChecksControllerTests.cs]: This appears to only cover one of the three possible scenarios when looking up the website address

Comment (MakCom) [reply src/SFA.DAS.ApplyService.InternalApi.UnitTests/GatewayOrganisationChecksControllerTests.cs]: One more test scenario added

Comment (David-B-Read) [reply src/SFA.DAS.ApplyService.InternalApi.UnitTests/GatewayOrganisationChecksControllerTests.cs]: The code is handling/logging a message when no website is found - that was the third scenario I was referring to

Comment (MakCom) [reply src/SFA.DAS.ApplyService.InternalApi.UnitTests/GatewayOrganisationChecksControllerTests.cs]: All scenarios now covered.

File: src/SFA.DAS.ApplyService.InternalApi.UnitTests/GatewayOrganisationChecksControllerTests.cs [hunk 2]
```diff
@@ -117,5 +120,45 @@ public void get_website_address_manually_entered_returns_no_value_when_not_prese
...
+               .Setup(x => x.GetAnswerValue(_applicationId,
+                                           RoatpWorkflowSequenceIds.YourOrganisation,
+                                           RoatpWorkflowSectionIds.YourOrganisation.WhatYouWillNeed,
+                                           RoatpWorkflowPageIds.WebsiteManuallyEntered,
+                                           RoatpYourOrganisationQuestionIdConstants.WebsiteManuallyEntered)).ReturnsAsync(applyWebsite);    
+
+
+            var actualResult = _controller.GetOrganisationWebsiteAddress(_applicationId).Result;
+
+            switch (testCase)
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.ApplyService.InternalApi.UnitTests/GatewayOrganisationChecksControllerTests.cs]: This definitely shouldn't be here.  If we are checking the case number we should have 4 different tests.

Comment (MichaelYoung1981) [reply src/SFA.DAS.ApplyService.InternalApi.UnitTests/GatewayOrganisationChecksControllerTests.cs]: Obviously reuse all of the setup above tho, no need to copy it all to each test.

Comment (MakCom) [reply src/SFA.DAS.ApplyService.InternalApi.UnitTests/GatewayOrganisationChecksControllerTests.cs]: Tests are refactored.

**Pull request closed without merging**